modules/core/src/main/scala/sangria/ast/QueryAst.scala

```diff
@@ -498,6 +514,8 @@ case class Comment(text: String, location: Option[AstLocation] = None) extends A
 
 // Schema Definition
 
+/** @group typesystem
```

Comment:
What is this `@group`? is if the scaladoc?

Reply:
https://docs.scala-lang.org/overviews/scaladoc/for-library-authors.html#member-grouping-tags

Reply:
You can already see an example [here](https://www.javadoc.io/doc/org.sangria-graphql/sangria_2.13/latest/sangria/ast/index.html).